package.json

```diff
@@ -23,114 +23,113 @@
     "format": "prettier --write \"(src|tests|.storybook|types)/**/*.{js,jsx,ts,tsx,scss}\""
   },
   "dependencies": {
-    "@date-io/luxon": "^2.10.8",
-    "@material-ui/core": "^4.11.3",
-    "@material-ui/lab": "^4.0.0-alpha.57",
+    "@date-io/luxon": "^2.10.11",
+    "@material-ui/core": "^4.11.4",
+    "@material-ui/lab": "^4.0.0-alpha.58",
     "@material-ui/pickers": "4.0.0-alpha.12",
-    "@stripe/react-stripe-js": "^1.2.2",
-    "@stripe/stripe-js": "^1.11.0",
-    "@szhsin/react-menu": "^1.9.0",
+    "@stripe/react-stripe-js": "^1.4.1",
+    "@stripe/stripe-js": "^1.15.0",
+    "@szhsin/react-menu": "^1.10.1",
     "chart.js": "^2.9.4",
     "chartjs-plugin-datalabels": "^0.7.0",
-    "classnames": "^2.2.6",
-    "clipboard": "^2.0.6",
-    "formik": "2.2.6",
-    "framer-motion": "^3.2.1",
-    "lodash": "^4.17.20",
-    "luxon": "^1.25.0",
-    "notistack": "^1.0.3",
-    "postcss-focus-visible": "^5.0.0",
-    "react": "16.13.1",
+    "classnames": "^2.3.1",
+    "clipboard": "^2.0.8",
+    "formik": "^2.2.9",
+    "framer-motion": "^4.1.17",
```

Comment:
framer-motion got a major version bump. Did we test to make sure there were no breaking api changes there?